packages/wrangler/src/dev/miniflare.ts

```diff
@@ -408,7 +411,9 @@ export function buildMiniflareBindingOptions(config: MiniflareBindingsConfig): {
 	// Setup service bindings to external services
 	const serviceBindings: NonNullable<WorkerOptions["serviceBindings"]> = {
 		...config.serviceBindings,
+		...(config.experimentalAssets ? { ASSET_SERVER: "asset-server" } : {}),
```

Comment:
We'll only want to do this when there is no user worker (i.e. when we're using the no-op-assets-worker as our entrypoint). Can we add that check in now? Or do we want to tackle that later when we add in custom user-specified bindings from user workers to ASW2?

Reply:
I'll be handling this as part of WC-2435